frontend/src/app/testResults/viewResults/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { Location } from "react-router-dom";
+import moment from "moment";
+
+import { getParameterFromUrl } from "../../utils/url";
+
+export function getFiltersFromUrl(url: Location) {
+  const patientId = getParameterFromUrl("patientId", url);
+  const startDate = getParameterFromUrl("startDate", url);
+  const endDate = getParameterFromUrl("endDate", url);
+  const role = getParameterFromUrl("role", url);
+  const disease = getParameterFromUrl("disease", url);
+  const result = getParameterFromUrl("result", url);
+  const filterFacilityId = getParameterFromUrl("filterFacilityId", url);
+
+  return {
+    ...(patientId && { patientId: patientId }),
```

Comment:
if any of these const's aren't defined, would we be trying to apply `...` to a boolean since `param && {paramKey: param}` would give us back false? 

Worried that might [throw an error](https://stackoverflow.com/questions/38849613/spread-operator-issue-with-float-and-boolean-variable)

Reply:
This code is already [in the app](https://github.com/CDCgov/prime-simplereport/blob/b2ed514b64f9f3ce4f6b93b234499440eb97ba1c/frontend/src/app/testResults/TestResultsList.tsx#L648C1-L663C5) but I still went and double check the behavior and it does not break if any value is passed as `null`. It is totally safe!.
<img width="445" alt="Screenshot 2023-11-13 at 9 09 04 AM" src="https://github.com/CDCgov/prime-simplereport/assets/103958711/41f583e0-6699-496c-b9eb-03a54f776ebc">